internal/history/history.go

```diff
@@ -0,0 +1,59 @@
+package history
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/AckeeCZ/goproxie/internal/gcloud"
+	"github.com/AckeeCZ/goproxie/internal/kubectl"
+	"github.com/AckeeCZ/goproxie/internal/store"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const KeyCommands = "history.commands"
+
+func extractPodBaseName(pod *kubectl.Pod) string {
+	// TODO
+	return pod.Name
+}
+
+func StorePodProxy(projectId string, cluster *gcloud.Cluster, namespace string, pod *kubectl.Pod, localPort int, remotePort int) {
+
+	record := fmt.Sprintf("-project=%v -cluster=%v -namespace=%v -pod=%v -local_port=%v -proxy_type=pod", projectId, cluster.Name, namespace, extractPodBaseName(pod), localPort)
```

Comment:
What about the remote port? Do we want to set it as well?

Reply:
We do, but this have to be resolved first https://github.com/AckeeCZ/goproxie/issues/6